### DISCUSSION TITLE
US deaths 7-day avg is off

### DISCUSSION THREAD

**Discussion by marcopiani** [owid/covid-19-data#2592] [General]
Hello,

there seems to be an issue with US death data, perhaps due to how JHU or OWID has handled recent corrections by interpolating data. It seems to have affected only the 7-day avg. 
![coronavirus-data-explorer (12)](https://user-images.githubusercontent.com/18381804/162323905-cc6198a2-b618-4fd2-a67e-ee0c36440576.png)

Cheers,

Marco

**Comment by edomt** [comment]
Hi Marco

Thanks - it's really strange, it seems that JHU has changed their entire time series for the US, and now every week since mid-2021 includes a small decrement in cumulative deaths:

<img width="965" alt="image" src="https://user-images.githubusercontent.com/7103774/162371264-bff23da0-cb97-4902-8251-bf0a14dc93c2.png">

This has indeed completely messed up the 7-day rolling average, because of how we calculate it in the presence of negative values. I've now adjusted that to make it more usable.

![coronavirus-data-explorer(1)](https://user-images.githubusercontent.com/7103774/162372184-768d0eaa-759a-4aa1-86b5-b04486617b6e.png)

**Reply by marcopiani** [reply]
Thank you, Edouard.

It might or might not have to do with the fact that the CDC had located some error in the dashboard data and triggered some recalculations / change of method.

I do not know how JHU gets the US total, but it is clear that the reporting to the CDC has changed in time for some of those states that do not report by date of death (the majority)

Thanks once more for the prompt reply and action.

Marco

![image](https://user-images.githubusercontent.com/18381804/162487174-de452f03-1d75-46df-9188-99f6885e3b7b.png)
![image](https://user-images.githubusercontent.com/18381804/162487899-1e4e1af5-f1d7-4984-811a-f76edc6b54d2.png)
